package.json

```
@@ -61,7 +73,7 @@
   },
   "license": "ISC",
   "peerDependencies": {
-    "zod": "^3.24.1"
+    "zod": "^4.0.14"
```

I think it is safer to use both 3 and 4.
```suggestion
    "zod": "^3.25 || ^4.0.14"
```

---

I think so. Better using "^3.25.0 || ^4.0.0". See: https://zod.dev/library-authors
```
{
  "peerDependencies": {
    "zod": "^3.25.0 || ^4.0.0"
  }
}
```